cantools/database/can/formats/dbc.py

```diff
@@ -1021,6 +1021,14 @@ def get_signal_name(frame_id_dbc, name):
         except (KeyError, TypeError):
             return name
 
+    def get_signal_initial_value(frame_id_dbc, name):
+        signal_attributes = get_attributes(frame_id_dbc, name)
+
+        try:
+            return signal_attributes['GenSigStartValue'].value
+        except (KeyError, TypeError):
+            return None
+
```

Comment:
Are you intentionally ignoring the default attribute value?

Reply:
you are right I am not passing the default attribute value, let me correct it

Reply:
I was looking at the Documentation and if the signal has no initial value a None should be returned, that is why I was returning the None if the signal has no GenSigStartValue attribute defined in the DBC file,  the default attribute value that you are referring is the default value from the DBC file?

Reply:
I'm not really sure myself how this should work. But yes, I was referring to the default value of the GenSigStartValue attribute in the DBC-file. I was just curious if the default value should be used or not, if you had even considered it.

I'm fine with only adding initial values to signals that have a GenSigStartValue attribute in the DBC-file, and give all other signals the initial value ``None``, just as you already implemented. We can change the behavior later if necessary.

Reply:
Yes I considered it, actually I think is not difficult to add that implementation, but I think it can be added later if it is really necessary, right now I am also good with this approach.